### ISSUE TITLE
REST API does not have light which is clearly shown in deConz UI

### ISSUE THREAD

**Issue by donnib** [dresden-elektronik/deconz-rest-plugin#132]
I don't know how this happened but i see the light in deConz but it does not exist in the REST API

**Comment by manup** [comment]
Have you tried to power cycle the light?

**Comment by donnib** [comment]
Nope, but should that be necessary ? I'll try that when i get home.

**Comment by manup** [comment]
It may help, normally lights once they have been discovered are available in the REST API instantly since they will be restored from cache.

Something seems to be messed up here, do the other lights work and have their correct names?

**Comment by donnib** [comment]
Yes they work and have the correct names. Only these two are messed up.
